db/migrate/20230426150134_create_organisations.rb

```diff
@@ -0,0 +1,24 @@
+class CreateOrganisations < ActiveRecord::Migration[7.0]
+  def change
+    create_table :organisations do |t|
+      t.string :content_id, null: false
+      t.string :slug, null: false
+      t.string :name, null: false
+
+      t.timestamps
+    end
+
+    add_index :organisations, :content_id, unique: true
+    add_index :organisations, :slug, unique: true
+
+    reversible do |dir|
+      dir.up do
+        Organisation.create(content_id: "af07d5a5-df63-4ddc-9383-6a666845ebe9", slug: "government-digital-service", name: "Government Digital Service")
+      end
+
+      dir.down do
+        Organisation.delete_all
+      end
+    end
```

Comment:
Thanks for splitting up the other migration. Could you do the same here?

Reply:
Done.